Index: wp-admin/includes/class-wp-list-table.php
===================================================================
--- wp-admin/includes/class-wp-list-table.php	(revision 35773)
+++ wp-admin/includes/class-wp-list-table.php	(working copy)
@@ -459,6 +459,10 @@
 		if ( empty( $this->_actions ) )
 			return;

+		if ( '2' === $two && 'plugins' === $this->screen->id ) {
+			return;
+		}
+
 		echo '<label for="bulk-action-selector-' . esc_attr( $which ) . '" class="screen-reader-text">' . __( 'Select bulk action' ) . '</label>';
 		echo '<select name="action' . $two . '" id="bulk-action-selector-' . esc_attr( $which ) . "\">\n";
 		echo '<option value="-1">' . __( 'Bulk Actions' ) . "</option>\n";
Index: src/wp-admin/js/updates.js
===================================================================
--- src/wp-admin/js/updates.js	(revision 36084)
+++ src/wp-admin/js/updates.js	(working copy)
@@ -524,6 +524,11 @@
 			e.preventDefault();
 			var $button = $( e.target );

+			// Do nothing if button disabled.
+			if ( $button.hasClass( 'button-disabled' ) ) {
+				return;
+			}
+
 			if ( wp.updates.shouldRequestFilesystemCredentials && ! wp.updates.updateLock ) {
 				wp.updates.requestFilesystemCredentials( e );
 			}
